data/json/mutations/mutations.json

```
@@ -5889,6 +5901,110 @@
     "threshold": true,
     "mutagen_target_modifier": 10
   },
+  {
+    "type": "mutation",
+    "id": "THRESH_RABBIT",
+    "name": { "str": "Lagomorph" },
+    "points": 1,
+    "description": "You really want to eat some carrots.  Welcome to the Lagomorpha family.",
+    "valid": false,
+    "purifiable": false,
+    "threshold": true
```

```suggestion
    "threshold": true,
    "mutagen_target_modifier": 10
```
Looks like the other (normal at least) threshold mutations use this to influence the balance it shoots for, so probably a good idea to use it here too